azurelinuxagent/ga/exthandlers.py

```diff
@@ -1387,6 +1391,17 @@ def get_env_file(self):
     def get_log_dir(self):
         return os.path.join(conf.get_ext_log_dir(), self.ext_handler.name)
 
+    def get_seq_no(self):
+        runtime_settings = self.ext_handler.properties.extensions
+        # If no runtime_settings available for this ext_handler, then return 0 (this is the behavior we follow
+        # for update_settings)
```

Comment:
could we refactor this into a function (get_feault_seq_no?) and call it here and in update_settings?

Reply:
discussed offline; good as it is; my bad